ingest-app/docs/api.md

```diff
@@ -581,40 +579,43 @@ The following update types are supported:
   * Find and update - merge update value into any instance in the list that matches the find value.
   * Find and remove - remove any instance from the list that matches the find value.
 
-Update types that include a find will match on the fields supplied. For example, for a science keyword update with a find value of {"Category": "EARTH SCIENCES"}, any science keyword with a category of "EARTH SCIENCES" will be considered a match regardless of the values of the science keyword topic, term, etc.
+Bulk update post request takes the following parameters:
+  
+  * Concept-ids (required) - a list of concept ids to update
+  * Update type (required) - choose from the enumeration: ADD_TO_EXISTING, CLEAR_ALL_AND_REPLACE, FIND_AND_REPLACE, FIND_AND_REMOVE, FIND_AND_UPDATE
+  * Update field (required) - choose from the enumeration: SCIENCE_KEYWORDS, LOCATION_KEYWORDS, DATA_CENTERS, PLATFORMS, INSTRUMENTS
+  * Update value (required for all update types except for FIND_AND_REMOVE) - UMM-JSON representation of the update to make
+  * Find value (required for FIND_AND_REPLACE, FIND_AND_UPDATE and FIND_AND_REMOVE update types) - UMM-JSON representation of the data to find
+
+Update types that include a FIND will match on the fields supplied in the find value. For example, for a science keyword update with a find value of {"Category": "EARTH SCIENCES"}, any science keyword with a category of "EARTH SCIENCES" will be considered a match regardless of the values of the science keyword topic, term, etc.  It's worth noting that find value can not contain nested fields. So for bulk update on PLATFORMS, for example, find value can only contain Type, ShortName and LongName, not the nested fields like Characteristics and Instruments. On the other hand, update value can contain all the valid fields including the nested fields. So, nested fields can be updated, they just can't be used to find the matches. update value can not contain arrays. 
```

Comment:
Can you expand on "update value cannot contain arrays"? How would that be accomplished then - do they call update-value multiple times to add each value they want? Also capitalize update.

Reply:
Yes. If you want to ADD_TO_EXISTING, with a list of platforms, you will have to add one at a time. If you want to CLEAR_ALL_AND_REPLACE, you can only replace with one platform.   DONE.